ui/StatusQ/src/StatusQ/Components/StatusCommunityCard.qml

```diff
@@ -37,6 +37,26 @@ import StatusQ.Core.Theme 0.1
 Rectangle {
     id: root
 
+    property int cardSize: StatusCommunityCard.Size.Big
```

Comment:
Missing the `You're here too` state.

<img width="247" alt="Screenshot 2024-03-01 at 09 42 50" src="https://github.com/status-im/status-desktop/assets/47811206/be970ec7-26a5-40f8-afc0-2e977606e00a">

Reply:
I believe this should be gone?